lib/vdsm/storage/lvm.py

```diff
@@ -913,8 +917,12 @@ def getAllVgs(self):
                 self.stats.miss()
                 for name in stalevgs:
                     del vgs[name]
-                reloaded = self._reloadvgs(stalevgs)
-                vgs.update(reloaded)
+                try:
+                    reloaded = self._reloadvgs(stalevgs)
+                    vgs.update(reloaded)
+                except se.VolumeGroupDoesNotExist:
+                    # LVM error prevents reload and return stale VGs
+                    pass
```

Comment:
This again must be done before or together with the change raising in _reloadvgs().

We cannot have a change that breaks some callers and fix this in later commits.